dev/com.ibm.ws.security.kerberos.auth/resources/com/ibm/ws/security/kerberos/auth/internal/resources/KerberosMessages.nlsprops

```diff
@@ -0,0 +1,27 @@
+###############################################################################
+# Copyright (c) 2020 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+###############################################################################
+#CMVCPATHNAME com.ibm.ws.security.kerberos.auth/resources/com/ibm/ws/security/kerberos/auth/internal/resources/KerberosMessages.nlsprops
+#COMPONENTPREFIX CWWKS
+#COMPONENTNAMEFOR WebSphere Application Server Security Kerberos Service
+#ISMESSAGEFILE TRUE
+#NLS_MESSAGEFORMAT_VAR
+#NLS_ENCODING=UNICODE
+# -------------------------------------------------------------------------------------------------
+
+# For any new messages reserve a code from the nlsprops file in the com.ibm.ws.security.kerberos.java8 bundle
+
+KRB5_FILE_NOT_FOUND_CWWKS4345E=CWWKS4345E: The attribute [{0}] from the {1} element is configured to a file that does not exist at: {2}
+KRB5_FILE_NOT_FOUND_CWWKS4345E.explanation=The configuration refers to a file that does not exist, or could not be read. Ensure that the configuration points to a file that exists and is readable by the appliction process.
+KRB5_FILE_NOT_FOUND_CWWKS4345E.useraction=Ensure that the configuration points to a file that exists and is readable by the application process.
+
+KRB5_FILE_FOUND_CWWKS4346I=CWWKS4346I: The Kerberos component is configured to use a {0} located at {1}
```

Comment:
What kind of elements are {0} and {1} ?

Reply:
{0} will be either a .keytab or .conf file. 
{1} will be the path to the configured keytab or conf file.
Since these bits won't be translated, I decided to use {0} for 1 less message that will need to be translated

Reply:
Generally, guidance is to have an identifier for the variable, like:

`The Kerberos component is configured to use a {0} file located at {1}.`

If the variable pulls in the actual filename

`The Kerberos component is configured to use a the {0} file located at {1}.`

The sentence already identifies {1} as a location so I don't think you'd need another identifier

Reply:
Generally, guidance is to include an identifier for variables, like:

The Kerberos component is configured to use a {0} located at {1}